hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/DBCheckpoint.java

```diff
@@ -55,4 +55,20 @@
    */
   void cleanupCheckpoint() throws IOException;
 
+  /**
+   * Set the OM Ratis snapshot index corresponding to the OM DB checkpoint.
+   * The snapshot index is the latest snapshot index saved by ratis
+   * snapshots. It is not guaranteed to be the last ratis index applied to
+   * the OM DB state.
+   * @param omRatisSnapshotIndex the saved ratis snapshot index
+   */
+  void setRatisSnapshotIndex(long omRatisSnapshotIndex);
```

Comment:
I think this can be called lastTransactionAppliedIndex, so that it will be easy to understand.